files/build_templates/database.service.j2

```diff
@@ -2,6 +2,7 @@
 Description=Database container
 Requires=docker.service
 After=docker.service
+After=rc-local.service
```

Comment:
>rc-local.service [](start = 6, length = 16)

I think you plan to use rc-local to migrate database cross image version. I believe this logic is the responsible of database service in the new image.

BTW, since database changes happen just few times, we may limited what we can warm-reboot/fast-reboot to different versions. For example, if database changes at version 0.19->0.20, then we ask for an upgrade path like 0.1x->0.19->0.20->0.2X, not a random 0.1x->0.2x

Reply:
> just want to make sure the config file at /etc/sonic is copied before database docker start to use it

You mean this is a bug fix? If yes, then I am ok.

Reply:
yes If we want to use something under /etc/sonic/, first we need to have it which is done in rc.local. But for us,  it is OK now, since the database config is static as mentioned, we can always copy it from the image at the very beginning, we don't need to restore the file from other version.